src/doc/unstable-book/src/language-features/auto-traits.md

```diff
@@ -1,106 +0,0 @@
-# `auto_traits`
```

Comment:
leaving a comment here to remember to copy these docs to somewhere else. maybe we should have a subdirectory about rustc_attrs in the unstable book?